Python/Tests/ProfilingTests/ProfilingTests.cs

```diff
@@ -57,9 +57,8 @@ public class ProfilingTests {
                 Assert.IsTrue(File.Exists(testFile), "Did not find " + testFile);
             }
 
-            // Test in 3.4 for tokenize.open and 3.1 for tokenize.detect_encoding
             // Python 2.x uses execfile() and we do not handle encoding at all
-            foreach (var python in new[] { PythonPaths.Python31, PythonPaths.Python34 }) {
+            foreach (var python in new[] { PythonPaths.Python37, PythonPaths.Python36 }) {
```

Comment:
These tests were for functionality related to those particular versions. There should be other tests covering 3.7 and 3.6 already, so this whole test can probably just be removed.

Reply:
This is the only test case for this scenario. I have commented it out and will revisit it when I fix the broken tests.